packages/video_player/video_player/test/video_player_test.dart

```diff
@@ -754,7 +776,32 @@ void main() {
     });
 
     group('caption', () {
-      test('works when seeking', () async {
+      test('makes sure the input captions are unsorted', () async {
```

Comment:
Why is this a unit test? I.e., why is asserting that the visible caption list is not sorted useful? What user behavior would break if this test were failing?

Reply:
the idea is this test makes sure that the input captions were unsorted, and the tests after it gets the right caption

I am just making sure that no one in the future resorting the input captions, and someone else removing the sorting from inside of the video player in that case the binary search will be broken for unsorted inputs.

please correct me if my thought process is wrong.